### DISCUSSION TITLE
I have an issue during installation with masons plugins

### DISCUSSION THREAD

**Discussion by rustydwemer** [NvChad/NvChad#2053] [General]
The  following error rises:

spawn: npm.cmd failed with exit code - and signal -. npm.cmd is not executable

for the following plugins: 

typescript-language-server
prettier
css-lsp
html-lsp

I am on windows 10 and I have installed all the prerequisites listed at https://nvchad.com/docs/quickstart/install . Note that I am new to nvim/vim and I mainly used Visual Studio.

It also appears that the list there is not full: during installation of nvchad, I also run into the issue that it wanted python3 and path to python 3 in the Path sys variable. I have fixed the python issue. But the npm.cmd error stays. 

Do I need to install some additional dependencies? 

And what the purpose of the those 4 plugins? Are those just syntax highlight for typescrip/css/html? 

As of now I want to use nvim with rust (I will attempt to configure that myself). So can I just ignore the error I get and be sure that all the features of nvchad would work as expected?

**Comment by siduck** [comment]
@rustydwemer  install nodejs + npm
